src/odemis/acq/stitching/test/weaver_test.py

```diff
@@ -337,7 +288,7 @@ def test_gradient(self):
         outd = weaver.getFullImage()
 
         # Visualize overlap mask
-        #from PIL import Image
+        # from PIL import Image
         # Image.fromarray(outd).show()
```

Comment:
Since it is suggested to look at the output the expected results should alse be drawn somewhere.
I think some basic ASCI art should work. There are converters on the internet for JPG (etc) to ASCI art txt.

Reply:
I can't get a good ascii art visualization, I'll update the comment so it at least explains what it should look like.